numpy/ma/core.py

```diff
@@ -5213,7 +5213,8 @@ def round(self, decimals=0, out=None):
             out.__setmask__(self._mask)
         return out
 
-    def argsort(self, axis=None, kind='quicksort', order=None, fill_value=None):
+    def argsort(self, axis=-1, kind='quicksort', order=None,
+                endwith=True, fill_value=None):
```

Comment:
Also, `argsort` is documented as having a default argument of `-1`, even though the actual default is `None`. So can we update the API to match the documentation?